omod/src/test/java/org/openmrs/module/fhir2/providers/EncounterFhirResourceProviderWebTest.java

```diff
@@ -82,31 +125,145 @@ public void getEncounterByWrongUuid_shouldReturn404() throws Exception {
 	}
 	
 	@Test
-	public void findEncountersByPatientIdentifier_shouldReturnBundleOfEncounters() throws IOException, ServletException {
-		Encounter encounter = new Encounter();
-		encounter.setId(ENCOUNTER_UUID);
-		when(encounterService.findEncountersByPatientIdentifier(PATIENT_IDENTIFIER))
-		        .thenReturn(Collections.singletonList(encounter));
+	public void shouldGetObservationsBySubjectUuid() throws Exception {
```

Comment:
Observations?

Reply:
@ibacher Oh sorry was a bit lazy in typing the full name so copied it from Observations and then forgot to change it to encounters.

Reply:
@VaishSiddharth That happens and it's fine; just needs to be fixed before we can merge it 🙂